cmd/state/state.go

```diff
@@ -193,5 +203,17 @@ func getFlags(defaultFlags GlobalFlags, env map[string]string) GlobalFlags {
 	if _, ok := env["K6_PROFILING_ENABLED"]; ok {
 		result.ProfilingEnabled = true
 	}
+	if env["K6_BINARY_PROVISIONING"] == "true" {
```

Comment:
For consistency with other "flags", like `K6_PROFILING_ENABLED` right above, I'd suggest to use the presence of that environment variable as the condition to consider this feature to be enabled.

Or, in case we consider that risky, I'd at least accept any "truthy" value there. Not that it's not only that `K6_BINARY_PROVISIONING=1` won't enable it, but nor `K6_BINARY_PROVISIONING=True` or `K6_BINARY_PROVISIONING=TRUE` would, which in my humble opinion isn't very good user experience.

Reply:
You can use `strconv.ParseBool` as reference/inspiration:
https://pkg.go.dev/strconv#ParseBool